vllm/config.py

```diff
@@ -542,18 +542,19 @@ class SchedulerConfig:
             prompt latency) before scheduling next prompt.
         enable_chunked_prefill: If True, prefill requests can be chunked based
             on the remaining max_num_batched_tokens.
+        preemption_mode: Whether to perform preemption by swapping or 
+            recomputation (default)
```

Comment:
this comment isn't exactly right, seems the scheduler can dynamically choose the mode if not specified

`seq_group.get_max_num_running_seqs() == 1:`